src/assets/style.css

```diff
@@ -2,6 +2,14 @@ tr.first-bracket:not(:first-child) > td {
   border-top-width: 5px;
 }
 
+p.descriptionForVariables{
+  font-size: large;
+  margin-top: 3%;
+}
+span.variableMetadataHighlight{
```

Comment:
Also, if you just want to make some text bold, the [native `strong` tag](https://developer.mozilla.org/fr/docs/Web/HTML/Element/strong) should be enough.